drivers/user_permission_report/app/views/user_permission_report/warehouse_reports/reports/index.haml

```diff
@@ -44,10 +44,13 @@
                   - else
                     = render 'admin/inactive_users/user_status', user: user
                 %br
-                - if user.last_sign_in_at.present?
-                  Last login: #{user.last_sign_in_at}
```

Comment:
this was showing the "last_sign_in_at" which is the login from their PREVIOUS session, not their most recent session. I assume that was unintentional so updated it to show the most recent login instead